packages/quantic/force-app/main/default/lwc/quanticResultPrintableUri/__tests__/quanticResultPrintableUri.test.js

```diff
@@ -0,0 +1,149 @@
+// @ts-ignore
+import {
+  // @ts-ignore
+  getNavigateCalledWith, // @ts-ignore
+  getGenerateUrlCalledWith,
+} from 'lightning/navigation';
```

Comment:
```suggestion
// @ts-ignore
import { getNavigateCalledWith, getGenerateUrlCalledWith } from 'lightning/navigation';
```

Reply:
That's the linter's work, we can't really change that.

Reply:
how weird x)